src/ape/managers/project/manager.py

```diff
@@ -95,6 +95,16 @@ def contracts_folder(self) -> Path:
 
         return folder
 
+    @property
+    def compiler_cache_folder(self) -> Path:
+        """
+        The path to the project's compiler source cache folder.
+        """
+        # NOTE: as long as config has come up properly, this should not be None
+        if self.config_manager.compiler_cache_folder is None:
+            raise ProjectError("Compiler cache folder not set in config.")
```

Comment:
concern: this error message implies you must configure the cache folder even though that is not the case.